src/components/Tile/Tile.styles.ts

```diff
@@ -0,0 +1,43 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import { memoizeFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { mergeStyleSets, ITheme } from 'office-ui-fabric-react/lib/Styling';
+
+export interface TileClassNameProps {
+  theme: ITheme;
+}
+
+export const getClassNames = memoizeFunction((classNameProps: TileClassNameProps) => {
+  const { theme } = classNameProps;
+
+  return mergeStyleSets({
+    root: {
+      border: `1px solid ${theme.semanticColors.bodyDivider}`,
+      selectors: {
+        '&.y-tile': {
+          position: 'relative',
+          width: '100%',
+          backgroundColor: theme.semanticColors.bodyDivider,
```

Comment:
Is there a reason not to move these 3 up to the root?

Reply:
Ah, no reason. I may have copied something I saw in `Text`.